glyph/src/ttfp.rs

```diff
@@ -56,7 +123,7 @@ impl<'a> From<ttfp::RasterGlyphImage<'a>> for GlyphImage<'a> {
 /// Valid formats for a [`GlyphImage`].
 // Possible future formats:  SVG, JPEG, TIFF
 #[non_exhaustive]
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, Copy)]
```

Comment:
Maybe better not to add `Copy` as it restricts what new variants we can add later